variables.tf

```diff
@@ -49,6 +49,18 @@ variable "aws_kms_key_arn" {
   description = "The ARN of the AWS Key Management Service (AWS KMS) customer master key (CMK) to use to encrypt the EBS volumes"
 }
 
+variable "aws_ec2_image_block_public_access" {
+  type        = bool
+  default     = true
+  description = "Set to true to regionally block new AMIs from being publicly shared"
+}
+
+variable "aws_ebs_snapshot_block_public_access" {
+  type        = string
+  default     = "block-new-sharing"
+  description = "Configure regionally the EBS snapshot public sharing policy, alternatives: `block-all-sharing` and `unblocked`"
```

Comment:
One more thing could you add validation for this since only a few values are allowed? e.g. something like this without the null check https://github.com/schubergphilis/terraform-aws-mcaf-s3/blob/master/variables.tf#L317C1-L317C15